Source/4_0/CentralManager.swift

```diff
@@ -0,0 +1,47 @@
+import Foundation
+import RxSwift
+import CoreBluetooth
+
+public final class CentralManager: CentralManagerType {
+
+    public let cbCentralManager: CBCentralManager
+
+    init(cbCentralManager: CBCentralManager) {
+        self.cbCentralManager = cbCentralManager
+    }
+
+    /// Observable which emits state changes of central manager after subscriptions
+    var rx_didUpdateState: Observable<BluetoothState> {
+        return .empty()
+    }
```

Comment:
I think that this is opened question. Added TODO and we should handle this in future and make some kind of guidelines. We have a couple of reactive values exposed to the users, and probably using underscore is not the best idea in Swift 3. Also, the problem is that `Reactive` is part of `RxCocoa`, not `RxSwift`.